teku/src/main/java/tech/pegasys/teku/cli/options/ExecutionEngineOptions.java

```diff
@@ -27,7 +27,16 @@
       hidden = true)
   private String executionEngineEndpoint = null;
 
+  @Option(
+      names = {"--Xee-version"},
+      paramLabel = "<STRING>",
+      description = "Execution Engine API version. Possible values are: kintsugi (default) or kiln",
+      arity = "1",
+      hidden = true)
+  private String executionEngineVersion = null;
```

Comment:
Ideally this would be `private Version executionEngineVersion = Version.DEFAULT_VERSION`.  Then PicoCLI will take care of validation, do case insensitive matching and display the default value.  And we then wouldn't need to use `Optional<Version>` through the code and could just require a value always be set.

Reply:
ah yes, I was not aware of that. switching.